pages/fpki_intro.md

```diff
@@ -31,6 +29,7 @@ The participating Certification Authorities and the policies, processes, and aud
 
 The Federal PKI includes U.S. Federal, State, Local, Tribal, Territorial, and International Governments, as well as commercial organizations, that work together to provide services for the benefit of the Federal Government.
 
+Use the [FPKI Graph](https://fpki.idmanagement.gov/) to see the relationships between the Certification Authorities in the Federal PKI ecosystem. It graphically depicts how each Certification Authority links to another, through cross-certificates, subordinate certificates, or Bridge CAs.

```

Comment:
should we add the direct link? this links to top level / may be circular for a user...(on page intro, links back to intro!)

Reply:
Agree!